packages/app/src/app/components/Preview/DevTools/index.tsx

```diff
@@ -81,7 +80,6 @@ const VIEWS: IViews = {
   [tests.id]: tests,
   [terminal.id]: terminal,
   [reactDevTools.id]: reactDevTools,
-  [terminalUpgrade.id]: terminalUpgrade,
```

Comment:
This shouldn't be imported here as it is used on the `embed` routes and Overmind is not setup. Instead there is an `addedViews` prop for this purpose.

Reply:
nice!